requirements/_tests.txt

```diff
@@ -15,4 +15,5 @@ requests <=2.31.0
 fire <=0.5.0
 
 cloudpickle >1.3, <=3.0.0
-scikit-learn >=1.1.1, <1.4.0
+scikit-learn >=1.1.1, <1.3.0; python_version < "3.9"
+scikit-learn >=1.4.0, <1.5.0; python_version >= "3.9"
```

Comment:
shouldn't this be
```suggestion
scikit-learn >=1.3.0, <1.5.0; python_version >= "3.9"
```
otherwise versions in the interval `[1.3.0, 1.4.0)` are omitted.
assuming 1.3.0 breaks python 3.8 functionality. 

if 1.4.0 breaks it, then the change would be

```
scikit-learn >=1.1.1, <1.4.0; python_version < "3.9"
scikit-learn >=1.4.0, <1.5.0; python_version >= "3.9"
```

Reply:
The same as https://github.com/Lightning-AI/torchmetrics/pull/2198/files#r1508408419